src/bento/dashboardTabData.js

```diff
@@ -1206,21 +1192,21 @@ export const tabContainers = [
         tooltipText: 'sort',
         role: cellTypes.DISPLAY,
       },
-      // Extra Hidden Columns
       {
-        dataField: 'diagnosis_id',
-        header: 'Diagnosis ID',
-        display: false,
+        dataField: 'anatomic_site',
+        header: 'Anatomic Site',
+        display: true,
         tooltipText: 'sort',
         role: cellTypes.DISPLAY,
       },
       {
-        dataField: 'diagnosis_comment',
-        header: 'Diagnosis Comment',
-        display: false,
+        dataField: 'age_at_diagnosis',
+        header: 'Age at Diagnosis (days)',
+        display: true,
         tooltipText: 'sort',
         role: cellTypes.DISPLAY,
       },
+      // Extra Hidden Columns
```

Comment:
is this an extraneous comment?

Reply:
I think the comment gives a hint that a hidden column can be added. As there is another way to do the same thing in bento, it gives clarity and avoids ambiguity.

Reply:
@huuaho @Senait7-ux Yes, I added that comment but now as I think now if we are re-arranging this comments would not be helpful. We can remove it.

Reply:
Okay, will remove it.

Reply:
Thanks @Senait7-ux and @huuaho